VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -67,10 +67,11 @@
                 app:iconColor="@color/edx_grayscale_neutral_base"
                 app:iconName="fa-thumb-tack" />
 
-            <include
-                layout="@layout/discussion_author_layout"
+            <include layout="@layout/discussion_author_layout" />
+
+            <View
```

Comment:
You could use `Space` instead of `View`, which removes all the unnecessary drawing logic, and is designed for this purpose.